src/commands/Moderation/Utilities/time.ts

```diff
@@ -96,6 +96,7 @@ export default class extends SkyraCommand {
 			case Moderation.TypeCodes.FastTemporaryVoiceMute:
 			case Moderation.TypeCodes.TemporaryVoiceMute:
 			case Moderation.TypeCodes.VoiceMute: return this.checkVMute(message, user);
+			// TODO: Add the rest of restrictions
```

Comment:
Is this PR still a WIP?

Reply:
Technically, yes, since this is missing. But it should be simple enough for anybody else to pick it up, which is why I didn't do `// TODO(kyranet)`.

I want to work on other fixes and this is too tedious.